backend/tests/integration-async/migrator/test_migrator_tasks.py

```diff
@@ -662,3 +664,151 @@ def test_update_relations_with_modified_experiments(
     ).one()
 
     assert experiment_literature_relation.literature_uuid == record.id
+
+
+def test_disambiguation_runs_after_record_creation(
+    inspire_app, celery_app_with_context, celery_session_worker
+):
+    author_data = faker.record("aut", with_control_number=True)
+    author_data.update(
+        {
+            "name": {"value": "Brian Gross"},
+            "ids": [{"schema": "INSPIRE BAI", "value": "J.M.Maldacena.1"},],
+            "email_addresses": [{"current": True, "value": "test@test.com"}],
+        }
+    )
+    author_record = InspireRecord.create(author_data)
+    db.session.commit()
+    time.sleep(5)
```

Comment:
you shouldn't use a `sleep` here, but look at @harunurhan's way (I don't remember the details now, ask him if you don't find it).